### PR TITLE
tests: new helper to resize the ubuntu-seed partiition in nested tests

### PR DESCRIPTION
**Pull request by sergiocazzolato**
This new helper fixes 2 nested tests:
 1. core20-remodel
 2. core20-create-recovery (moved from core to manual suite)

Tests can add an env var called NESTED_UBUNTU_SEED_SIZE with the desired size.

### CODE REVIEW

**Review by Meulengracht — APPROVED** [review]
thank you!

File: tests/lib/tweak-gadget.py
Comment (alfonsosanchezbeato) [inline tests/lib/tweak-gadget.py:1]: Do we really want to remove this file? It is run from `mkimage-uc22` and `fde-on-classic`. `manip_ubuntu_seed.py` is not doing the same changes either.

**Review by alfonsosanchezbeato — COMMENTED** [review]
This looks good, but I have a question about the removed script

File: tests/lib/tweak-gadget.py
Comment (sergiocazzolato) [reply tests/lib/tweak-gadget.py:1]: right, I see mkimage-uc22 is using it but fde-on-classic is not
I'll revert that
Thanks for the review

**Review by alfonsosanchezbeato — APPROVED** [review]
LGTM, thanks

**Pull request merged**